Sources/URLUtils.swift

```diff
@@ -13,7 +13,7 @@ import AEPServices
 import Foundation
 
 /// Provides functions to append visitor information to a URL
```

Comment:
nit: comment needs updating as well, e.g.: Provides utility functions to generate the URL variables